lib/Image/Operation/Grayscale.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Timber\Image\Operation;
+
+use Timber\Helper;
+use Timber\ImageHelper;
+use Timber\Image\Operation as ImageOperation;
+
+/**
+ * Implements converting an image to black and white.
+ * Argument:
+ * - color to fill transparent zones
+ */
+class Grayscale extends ImageOperation {
+
+	private $quality;
+
+	/**
+	 * @param string $quality  ranges from 0 (worst quality, smaller file) to 100 (best quality, biggest file)
+	 */
+	public function __construct( $quality ) {
```

Comment:
I think this is the only big Q I have. The arg style we're using elsewhere is based on naming things in an associative array / object, rather than relying on arg order. Thus:

`{{ img.src | grayscale({quality:80}) }}`

... I think there could be some good questions about whether the grayscale filter _should_ consider quality (or if this should be handled by `| tojpg`, etc.). But given that this is how letterbox, resize, etc. all work let's start here.

Reply:
Agreed here. Removed.